pkg/test/matchers/matchers.go

```diff
@@ -2,7 +2,7 @@ package matchers
 
 import (
 	"github.com/google/go-cmp/cmp"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega"
```

Comment:
We use `.` in the import for `gomega` in most every test file, for ease of writing- no need to remove